### PR TITLE
Improve vignettes

### PR DESCRIPTION
**Pull request by jacobvjk**
this PR improves vignettes by
- correcting some typos
- correcting a formulation
- not using hard coded defaults

### CODE REVIEW

**Review by maurolepore — APPROVED** [review]
LGTM. In general, watch for duplicated documentation, e.g. the PDF manual, README, and Get started. Minimize duplication to make maintenance easier and to not overwhelm the reader.

File: vignettes/articles/run-stress-test.Rmd [hunk 1]
```diff
@@ -65,15 +64,15 @@ README.
...
-* lgd_senior_claims (0.45)
-* lgd_subordinated_claims (0.75)
-* terminal_value (0)
-* risk_free_rate (0.02)
-* discount_rate (0.02)
-* div_netprofit_prop_coef (1)
-* shock_year (2030)
-* term (2)
-* company_exclusion (TRUE)
+* lgd_senior_claims (`r formals(run_stress_test)$lgd_senior_claims`)
```
Comment (MirijaHa) [inline vignettes/articles/run-stress-test.Rmd:67]: nice 😎

**Review by MirijaHa — COMMENTED** [review]
LGTM, just please note that I already merged the PR to get rid of envvars. It might be a bit annoying to update the vignette etc. while this is still evolving (we will likely further simplify how data is passed to the function), but so close to the release we might want to make sure things stay in synch.

**Comment by jacobvjk** [comment]
> LGTM, just please note that I already merged the PR to get rid of envvars. It might be a bit annoying to update the vignette etc. while this is still evolving (we will likely further simplify how data is passed to the function), but so close to the release we might want to make sure things stay in synch.

explicit file paths as arguments are now added.

**Review by MirijaHa — APPROVED** [review]

**Pull request merged**